tests/test_date_parser.py

```diff
@@ -276,6 +276,56 @@ def test_dates_without_preference(self, date_string, expected):
         self.then_period_is('day')
         self.then_date_obj_exactly_is(expected)
 
+    @parameterized.expand([
+        param('February 2015', datetime(2015, 1, 31), datetime(2015, 2, 28)),
+        param('February 2012', datetime(2015, 1, 31), datetime(2012, 2, 29)),
+        param('March 2015', datetime(2015, 1, 25), datetime(2015, 3, 25)),
+        param('April 2015', datetime(2015, 1, 31), datetime(2015, 4, 30)),
+        param('April 2015', datetime(2015, 2, 28), datetime(2015, 4, 28)),
+        param('December 2014', datetime(2015, 2, 15), datetime(2014, 12, 15)),
```

Comment:
What do you say if we use keyword arguments for readability: `param('December 2014', today=datetime(2015, 2, 15), expected=datetime(2014, 12, 15))` ?